grails-app/services/grails/plugin/springsecurity/oauth2/SpringSecurityOauth2BaseService.groovy

```diff
@@ -61,7 +62,7 @@ class SpringSecurityOauth2BaseService {
      */
     String getAuthorizationUrl(String providerName) {
         OAuth2AbstractProviderService providerService = getProviderService(providerName)
-        providerService.getAuthUrl(['scope': _providerConfigurationMap.get(providerName).scope])
+        providerService.getAuthUrl([:])
```

Comment:
Is there any specific reason to remove the `scope` value?

Reply:
Yes, the configured scope is now cleanly passed (via an appropriate setter) to the `OAuth20Service` when building it:
https://github.com/grails-plugins/grails-spring-security-oauth2/pull/6/files#diff-0483ee04253e76f4d0406101ecad6e6eR117

The underlying Scribejava library will then properly forward it to the OAuth provider when building the authorization URL and when requesting the access token.

Thank you for merging this PR by the way !